nengo/spa/vocab.py

```diff
@@ -92,7 +92,7 @@ def create_pointer(self, attempts=100, unitary=False):
             count = 0
             p = pointer.SemanticPointer(self.dimensions, rng=self.rng)
             if self.vectors.shape[0] > 0:
-                while count < 100:
+                while count < attempts:
```

Comment:
Ok, I've updated this pull request with a new commit that includes Terry's suggestion. I checked that the best pointer is saved properly using a few print statements, but feel free to pass along any suggestions for improvements.